tools/install_travis.sh

```diff
@@ -54,10 +53,8 @@ else
     # Install lalsuite
 
     cd ${SRC}
-    git clone -q https://github.com/lscsoft/lalsuite.git
+    git clone -q https://github.com/ahnitz/lalsuite.git
     cd lalsuite
-    git checkout lalsuite_o1_branch
```

Comment:
True, but it's far easier to revert the whole thing and add back what
should be there.

On Tue, Sep 29, 2015 at 10:38 AM, spxiwh <notifications@github.com> wrote:

> In tools/install_travis.sh
> <https://github.com/ligo-cbc/pycbc/pull/397#discussion_r40646718>:
>
> >      cd lalsuite
> > -    git checkout lalsuite_o1_branch
>
> And I'm not sure about whether this one should be or not?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ligo-cbc/pycbc/pull/397/files#r40646718>.
>